profile/README.md

```diff
@@ -1,15 +1,41 @@
 # Think big
 
-![](https://archbee-image-uploads.s3.amazonaws.com/CGGF0VLgJQypTJbZ1FbQy/aoF2F0Tr5bORp7wnTzRec_github-org-page-banner.png)
+![](https://archbee-image-uploads.s3.amazonaws.com/CGGF0VLgJQypTJbZ1FbQy/aoF2F0Tr5bORp7wnTzRec_github-org-page-banner.png)  
+ 
+&nbsp;
+&nbsp;
+&nbsp;
+
+Voltage enables your team to create enterprise-grade experiences. By leveraging Voltage you receive the fastest and most scalable Lightning solution that meets the highest security and privacy standards.
+
+With modern modular tools and processes, you can build, deploy, and scale faster than ever.
+Voltage brings a suite of lightning node and data services to the market that elevates your value proposition through Lightning Network architecture.
+
+You are here to either setup enterprise grade Bitcoin infrastructure in less than two minutes, or seek further information on Voltage, so let’s get you where you need to be.
+
+&nbsp;
+
+[![X](https://github.com/LiamCYD/.github/blob/patch-1/X%20icon%20x2.png?raw=true)][1]    &nbsp;    [![linkedin](https://github.com/LiamCYD/.github/blob/patch-1/Linkedin%20icon%20x2.png?raw=true)][2]    &nbsp;    [![discord](https://github.com/LiamCYD/.github/blob/patch-1/Discord%20icon%20x2.png?raw=true)][3]
```

Comment:
Ideally we add any images for this page into this repo, preferably in a directory (although root of the repo will technically work). Right now the `x2` icons are in the root of your fork, and the others are under `profile`. If we put them all under `profile` then you should be able to reference them by filename-only, instead of specifying a complete URI.

Here you've requested to add them via this PR, however the links point to your fork instead. This allows them to be rendered in a preview, but it creates an unnecessary external dependency on that branch of your fork. If you delete that branch (which you should after this is merged) the images will disappear.

I would add them all in the `profile` dir (so move the `x2` images into there), and update image URIs like this:
```
[![X](X%20icon%20x2.png)][1] &nbsp; [![linkedin](Linkedin%20icon%20x2.png)][2] &nbsp; [![discord](Discord%20icon%20x2.png)][3]
```

That will allow them to work in a [preview](https://github.com/LiamCYD/.github/blob/patch-1/profile/README.md) and also continue working after merging, with no external dependencies.

Reply:
Appreciate it sir.